### PR TITLE
Feature/implement orm

### PR DESCRIPTION
**Pull request by AngelicaAgudelo**
Add the table models with sequalize ORM

-user
-service
-payment_service
-item
-payment_item

### CODE REVIEW

File: back-end-yaz/src/models/user.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        unique: true,
+      },
+      user_phone: {
+        type: DataTypes.STRING,
+      },
+      user_address: {
+        type: DataTypes.STRING,
+      },
+      user_address: {
+        type: DataTypes.STRING,
```
Comment (juanDuqueS) [inline back-end-yaz/src/models/user.js:36]: two user_address

File: back-end-yaz/src/models/user.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        allowNull: false,
+        unique: true,
+      },
+      user_phone: {
+        type: DataTypes.STRING,
+      },
+      user_address: {
+        type: DataTypes.STRING,
+      },
+      user_address: {
```
Comment (prometteo14) [inline back-end-yaz/src/models/user.js:35]: This atribute is already created

File: back-end-yaz/src/models/payment_service.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      id_service: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+      },
+      payment_date: {
+        type: DataTypes.DATE,
+        allowNull: false,
+      },
+    });
+
```
Comment (juanDuqueS) [inline back-end-yaz/src/models/payment_service.js:18]: format document required

**Review by juanDuqueS — CHANGES_REQUESTED** [review]
make changes

**Review by prometteo14 — CHANGES_REQUESTED** [review]

**Review by juanDuqueS — APPROVED** [review]
all good (y)

File: back-end-yaz/src/models/user.js (hunk 1 above)
Comment (AngelicaAgudelo) [reply back-end-yaz/src/models/user.js]: Removed, thanks!

File: back-end-yaz/src/models/user.js (hunk 2 above)
Comment (AngelicaAgudelo) [reply back-end-yaz/src/models/user.js]: Removed, thanks!

File: back-end-yaz/src/models/payment_service.js (hunk 3 above)
Comment (AngelicaAgudelo) [reply back-end-yaz/src/models/payment_service.js]: Fixed, thanks!

**Review by prometteo14 — APPROVED** [review]

**Review by alejandroVaron — APPROVED** [review]

**Pull request merged**